### PR TITLE
Mobile menu popup

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone : Portfolio - mobile menu
- Use basic JavaScript for side navigation bar.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.

### CODE REVIEW

**Comment by ibrahim777764** [comment]
Please review my code,thankyou

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```
Comment (keroloussamy) [inline .eslintrc.json:25]: - [ ] You did a great work adding the `eslintrc.json` file, but you should also update your `linters.yml` in the `.github/workflows` directory so GitHub can check it also like Html and CSS.

Please update your `linters.yml` file using this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file. 

As you see in the pic there is no check for `eslintrc`
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 

Good job so far!
There is one issue that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
updated yml file,hope so everything is completed as per requirement

**Review by keroloussamy — APPROVED** [review]
Hi @ibrahim777764, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**